From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Fri, 13 May 2016 11:13:16 +0800
Subject: Only test team devices if we built support for them.

Team device support is already conditional to teamdctl being available at
compile time. If we're not building support for them, we shouldn't attempt
to test them either as doing so would fail the build.
---
 src/platform/tests/test-link.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/platform/tests/test-link.c b/src/platform/tests/test-link.c
index ed43556..6395e02 100644
--- a/src/platform/tests/test-link.c
+++ b/src/platform/tests/test-link.c
@@ -2509,7 +2509,9 @@ _nmtstp_setup_tests (void)
 	g_test_add_func ("/link/internal", test_internal);
 	g_test_add_func ("/link/software/bridge", test_bridge);
 	g_test_add_func ("/link/software/bond", test_bond);
+#if WITH_TEAMDCTL
 	g_test_add_func ("/link/software/team", test_team);
+#endif
 	g_test_add_func ("/link/software/vlan", test_vlan);
 	g_test_add_func ("/link/software/bridge/addr", test_bridge_addr);
 
